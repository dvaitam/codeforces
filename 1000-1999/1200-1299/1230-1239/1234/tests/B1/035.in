10 5
3 6 7 9 9 1 6 9 7 9
