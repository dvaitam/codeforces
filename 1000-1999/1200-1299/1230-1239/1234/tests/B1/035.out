5
1 9 7 6 3
